### ISSUE TITLE
AttributeError: 'EPollReactor' object has no attribute '_handleSignals'

### ISSUE THREAD

**Issue by ShutdownRepo** [roys/cewler#1]
Running cewler raises the following error
```
# cewler --output wordlist.txt https://example.com


╭─────── CeWLeR v.1.1.1 - Custom Word List generator Redefined ────────╮
│                                                                      │
│   URL:           https://example.com                                 │
│   Strategy:      Exact same domain, max depth 2, 20 reqs/s           │
│   Words:         Mixed case, incl. numbers, min. 5 chars.            │
│   User-Agent:    Default (Mozilla/5.0 (Macintosh; Intel Mac OS...)   │
│   Output:        wordlist.txt                                        │
│   Time elapsed:  00:00                                               │
│   Requests:      -                                                   │
│   Data received: -                                                   │
│   Words found:   -                                                   │
│                                                                      │
╰──────────────────────────────────────────────────────────────────────╯
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /root/.local/pipx/venvs/cewler/lib/python3.11/site-packages/cewler/cewler.py:279 in cewler       │
│                                                                                                  │
│   276 │   │   │   process = CrawlerProcess(get_scrapy_settings_and_init_logging(args.user_agen   │
│   277 │   │   │   process.crawl(spider.CewlerSpider, console=console, url=args.url, file_words   │
│   278 │   │   │   │   │   │     without_numbers=args.without_numbers, min_word_length=args.min   │
│ ❱ 279 │   │   │   process.start()                                                                │
│   280 │   │   print("")                                                                          │
│   281 │   │                                                                                      │
│   282 │   │   if args.output is None:  # Should output to screen                                 │
│                                                                                                  │
│ /root/.local/pipx/venvs/cewler/lib/python3.11/site-packages/scrapy/crawler.py:376 in start       │
│                                                                                                  │
│   373 │   │   │   d.addBoth(self._stop_reactor)                                                  │
│   374 │   │                                                                                      │
│   375 │   │   if install_signal_handlers:                                                        │
│ ❱ 376 │   │   │   install_shutdown_handlers(self._signal_shutdown)                               │
│   377 │   │   resolver_class = load_object(self.settings["DNS_RESOLVER"])                        │
│   378 │   │   resolver = create_instance(resolver_class, self.settings, self, reactor=reactor)   │
│   379 │   │   resolver.install_on_reactor()                                                      │
│                                                                                                  │
│ /root/.local/pipx/venvs/cewler/lib/python3.11/site-packages/scrapy/utils/ossignal.py:19 in       │
│ install_shutdown_handlers                                                                        │
│                                                                                                  │
│   16 │   """                                                                                     │
│   17 │   from twisted.internet import reactor                                                    │
│   18 │                                                                                           │
│ ❱ 19 │   reactor._handleSignals()                                                                │
│   20 │   signal.signal(signal.SIGTERM, function)                                                 │
│   21 │   if signal.getsignal(signal.SIGINT) == signal.default_int_handler or override_sigint:    │
│   22 │   │   signal.signal(signal.SIGINT, function)                                              │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
AttributeError: 'EPollReactor' object has no attribute '_handleSignals'
```
# Env
pipx install (`pipx install cewler`)
pipx version 1.2.0, through Python 3.11.5
Debian 12, ARM64

**Comment by QU35T-code** [comment]
Update :

On debian11 it works fine but on debian12 there is an error :

```bash
cewler --output wordlist.txt https://example.com
Traceback (most recent call last):
  File "/root/.local/bin/cewler", line 5, in <module>
    from cewler.cewler import cewler
ImportError: cannot import name 'cewler' from 'cewler.cewler' (/root/.local/pipx/venvs/cewler/lib/python3.11/site-packages/cewler/cewler.py)
```

**Comment by n3rada** [comment]
Twisted shouldn't be part of the requirement tho'?

**Comment by QU35T-code** [comment]
I think that `Scrapy` dependency installs `Twisted` but the problem comes from the version of Twisted. That's why we need to set it to version `22.10.0`.

**Comment by roys** [comment]
Thank you for your report, @ShutdownRepo! @QU35T-code was right about regarding the dependency via Scrapy, and @ChrisAD gave me a nice PR fixing it in #2.

I've released a new version `1.1.2` to https://pypi.org/project/cewler/ . 

Please give me a word if you have any problems with it. :)

**Comment by ShutdownRepo** [comment]
Awesome news! Cewler will now be featured in [Exegol](https://exegol.rtfd.io)
